front-end/src/components/MostPopularVoucher.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+    <div id="popularVoucher">
+    <el-card id="chart-container">
+      <fusioncharts
+      :type="type"
+      :width="width"
+      :height="height"
+      :dataformat="dataFormat"
+      :dataSource="dataSource"
+      >
+      </fusioncharts>
+    </el-card>
+  </div>
+</template>
+
+<script>
+import Vue from 'vue';
+import VueFusionCharts from 'vue-fusioncharts';
+import FusionCharts from 'fusioncharts';
+import Charts from 'fusioncharts/fusioncharts.charts';
+import constants from "../constants"
+//import the theme
+import FusionTheme from 'fusioncharts/themes/fusioncharts.theme.fusion'
+
+// register VueFusionCharts component
+Vue.use(VueFusionCharts, FusionCharts, Charts, FusionTheme)
+
+const axios = require('axios').default;
+let api_server = 'http://localhost:3000';
+let scheme = '002'
```

Comment:
This value shouldn't be hard-coded. The chart might not work with other stores from our client